Sources/Commands/SwiftTool.swift

```diff
@@ -398,9 +398,11 @@ public class SwiftTool<Options: ToolOptions> {
         let packageRoot = findPackageRoot()
 
         self.packageRoot = packageRoot
-        self.buildPath = getEnvBuildPath() ??
+        
+        let buildPath = getEnvBuildPath(workingDir: cwd) ??
             customBuildPath ??
-            (packageRoot ?? localFileSystem.currentWorkingDirectory!).appending(component: ".build")
+            (packageRoot ?? cwd)
+        self.buildPath = buildPath.appending(component: ".build")
```

Comment:
The `.build` should only be appended in the last case, i.e. it should be `... (packageRoot ?? cwd).appending(component: ".build")`

Reply:
Make sense, thank you @aciidb0mb3r! :)